<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>dlgTrasnfoPtToPlot</class>
 <widget class="QDialog" name="dlgTrasnfoPtToPlot">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1088</width>
    <height>403</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Traiter les sommets nouveaux proches d'un sommet RFU</string>
  </property>
  <property name="styleSheet">
   <string notr="true">QTableView QHeaderView::section{
		border: 1px solid #ffffff;
		background-color: #818181;
		color: #ffffff;
    }
QGroupBox {
	margin-top: 16px;
	border: 1px solid gray;
	border-radius: 4px;
}
QGroupBox::title {
	subcontrol-origin: margin;
    subcontrol-position: top left;
	padding: 2px 5px 2px 5px;
	margin: 2px 0px 0px 0px;
	border:1px solid gray;
	border-radius: 2px;
	background: #696969;
	color: #ffffff;
}
	</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <property name="topMargin">
    <number>15</number>
   </property>
   <property name="verticalSpacing">
    <number>15</number>
   </property>
   <item row="0" column="0">
    <widget class="QGroupBox" name="nearvtxGp">
     <property name="enabled">
      <bool>true</bool>
     </property>
     <property name="title">
      <string>Sommets nouveaux proches d'un sommet RFU à traiter</string>
     </property>
     <layout class="QGridLayout" name="gridLayout_2">
      <property name="horizontalSpacing">
       <number>0</number>
      </property>
      <item row="3" column="0">
       <spacer name="horizontalSpacer">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeType">
         <enum>QSizePolicy::Fixed</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>20</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item row="3" column="1">
       <widget class="QLabel" name="nearvtxtrLab">
        <property name="styleSheet">
         <string notr="true">QLabel{
	border: 1px solid #ffffff;
	border-radius: 1px;
	background: #ff8b8b;
	padding: 0px 3px 0px 3px;
	color: #ffffff
}</string>
        </property>
        <property name="text">
         <string>Transformer ce sommet nouveau en détermination du sommet RFU proche : sélectionnez le sommet RFU à redéterminer</string>
        </property>
       </widget>
      </item>
      <item row="3" column="2">
       <widget class="QPushButton" name="selrfuvtxnearButt">
        <property name="enabled">
         <bool>true</bool>
        </property>
        <property name="sizePolicy">
         <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="minimumSize">
         <size>
          <width>24</width>
          <height>24</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>24</width>
          <height>24</height>
         </size>
        </property>
        <property name="text">
         <string/>
        </property>
        <property name="icon">
         <iconset>
          <normaloff>:/resources/select_rfuvtx</normaloff>:/resources/select_rfuvtx</iconset>
        </property>
        <property name="iconSize">
         <size>
          <width>24</width>
          <height>24</height>
         </size>
        </property>
        <property name="autoDefault">
         <bool>false</bool>
        </property>
        <property name="default">
         <bool>false</bool>
        </property>
        <property name="flat">
         <bool>false</bool>
        </property>
       </widget>
      </item>
      <item row="1" column="0" colspan="4">
       <widget class="QTableView" name="nearvtxTbv">
        <property name="frameShape">
         <enum>QFrame::NoFrame</enum>
        </property>
        <property name="alternatingRowColors">
         <bool>true</bool>
        </property>
        <property name="selectionMode">
         <enum>QAbstractItemView::SingleSelection</enum>
        </property>
        <property name="selectionBehavior">
         <enum>QAbstractItemView::SelectRows</enum>
        </property>
       </widget>
      </item>
      <item row="2" column="0" colspan="4">
       <widget class="QLabel" name="nearvtxLab2">
        <property name="text">
         <string>Action à réaliser sur le sommet nouveau cliqué:</string>
        </property>
       </widget>
      </item>
      <item row="0" column="0" colspan="4">
       <widget class="QLabel" name="nearvtxLab1">
        <property name="text">
         <string>Cliquez sur l'un des sommets nouveaux proches d'un sommet RFU ci-dessous puis choisissez l'action à réaliser</string>
        </property>
       </widget>
      </item>
      <item row="4" column="1">
       <widget class="QLabel" name="nearvtxltLab">
        <property name="styleSheet">
         <string notr="true">QLabel{
	border: 1px solid #ffffff;
	border-radius: 1px;
	background: #78c57d;
	padding: 0px 3px 0px 3px;
	color: #ffffff
}</string>
        </property>
        <property name="text">
         <string>Laisser ce sommet nouveau en l'état: cliquez sur le bouton pour valider cet action ne modifiant pas le sommet nouveau</string>
        </property>
       </widget>
      </item>
      <item row="4" column="2">
       <widget class="QPushButton" name="nearvtxvalButt">
        <property name="enabled">
         <bool>true</bool>
        </property>
        <property name="sizePolicy">
         <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="minimumSize">
         <size>
          <width>24</width>
          <height>24</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>24</width>
          <height>24</height>
         </size>
        </property>
        <property name="text">
         <string/>
        </property>
        <property name="icon">
         <iconset>
          <normaloff>:/resources/valid_nwvtx</normaloff>:/resources/valid_nwvtx</iconset>
        </property>
        <property name="iconSize">
         <size>
          <width>24</width>
          <height>24</height>
         </size>
        </property>
        <property name="autoDefault">
         <bool>false</bool>
        </property>
        <property name="default">
         <bool>false</bool>
        </property>
        <property name="flat">
         <bool>false</bool>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item row="2" column="0">
    <layout class="QHBoxLayout" name="butLay">
     <item>
      <widget class="QPushButton" name="buttResize">
       <property name="enabled">
        <bool>true</bool>
       </property>
       <property name="sizePolicy">
        <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="minimumSize">
        <size>
         <width>24</width>
         <height>24</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>24</width>
         <height>24</height>
        </size>
       </property>
       <property name="text">
        <string/>
       </property>
       <property name="icon">
        <iconset>
         <normaloff>:/resources/btn_resize</normaloff>:/resources/btn_resize</iconset>
       </property>
       <property name="iconSize">
        <size>
         <width>24</width>
         <height>24</height>
        </size>
       </property>
       <property name="autoDefault">
        <bool>false</bool>
       </property>
       <property name="default">
        <bool>false</bool>
       </property>
       <property name="flat">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="buttHsp">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="buttValid">
       <property name="text">
        <string>Quitter</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item row="1" column="0">
    <widget class="QGroupBox" name="farvtxGp">
     <property name="title">
      <string>Transformation forcée d'un sommet nouveau éloigné en détermination d'un sommet RFU</string>
     </property>
     <layout class="QGridLayout" name="gridLayout_3">
      <property name="horizontalSpacing">
       <number>0</number>
      </property>
      <item row="2" column="0">
       <spacer name="horizontalSpacer_2">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeType">
         <enum>QSizePolicy::Fixed</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>20</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item row="2" column="1">
       <widget class="QLabel" name="selnwvxtLab">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Fixed" vsizetype="Preferred">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="minimumSize">
         <size>
          <width>420</width>
          <height>0</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>420</width>
          <height>16777215</height>
         </size>
        </property>
        <property name="styleSheet">
         <string notr="true">QLabel{
	border: 1px solid #ffffff;
	border-radius: 1px;
	background: #79d7d4;
	padding: 0px 3px 0px 3px;
	color: #ffffff
}</string>
        </property>
        <property name="text">
         <string>1. Sélectionnez le sommet nouveau à transformer en détermination</string>
        </property>
       </widget>
      </item>
      <item row="3" column="1">
       <widget class="QLabel" name="selrfuvtxLab">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Fixed" vsizetype="Preferred">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="minimumSize">
         <size>
          <width>420</width>
          <height>0</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>420</width>
          <height>16777215</height>
         </size>
        </property>
        <property name="styleSheet">
         <string notr="true">QLabel{
	border: 1px solid #ffffff;
	border-radius: 1px;
	background: #ff8b8b;
	padding: 0px 3px 0px 3px;
	color: #ffffff
}</string>
        </property>
        <property name="text">
         <string>2. Sélectionnez le sommet RFU à redéterminer grâce au sommet nouveau</string>
        </property>
       </widget>
      </item>
      <item row="1" column="0" colspan="5">
       <widget class="QLabel" name="farvxtLab1">
        <property name="text">
         <string>Si vous souhaitez transformer un sommet nouveau en détermination d'un sommet RFU, bien que ce sommet nouveau ne se trouve pas dans la tolérance du sommet RFU:</string>
        </property>
       </widget>
      </item>
      <item row="2" column="2">
       <widget class="QPushButton" name="selnwvxtButt">
        <property name="enabled">
         <bool>true</bool>
        </property>
        <property name="sizePolicy">
         <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="minimumSize">
         <size>
          <width>24</width>
          <height>24</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>24</width>
          <height>24</height>
         </size>
        </property>
        <property name="text">
         <string/>
        </property>
        <property name="icon">
         <iconset>
          <normaloff>:/resources/select_nwvtx</normaloff>:/resources/select_nwvtx</iconset>
        </property>
        <property name="iconSize">
         <size>
          <width>24</width>
          <height>24</height>
         </size>
        </property>
        <property name="autoDefault">
         <bool>false</bool>
        </property>
        <property name="default">
         <bool>false</bool>
        </property>
        <property name="flat">
         <bool>false</bool>
        </property>
       </widget>
      </item>
      <item row="3" column="2">
       <widget class="QPushButton" name="selrfuvtxfarButt">
        <property name="enabled">
         <bool>true</bool>
        </property>
        <property name="sizePolicy">
         <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="minimumSize">
         <size>
          <width>24</width>
          <height>24</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>24</width>
          <height>24</height>
         </size>
        </property>
        <property name="text">
         <string/>
        </property>
        <property name="icon">
         <iconset>
          <normaloff>:/resources/select_rfuvtx</normaloff>:/resources/select_rfuvtx</iconset>
        </property>
        <property name="iconSize">
         <size>
          <width>24</width>
          <height>24</height>
         </size>
        </property>
        <property name="autoDefault">
         <bool>true</bool>
        </property>
        <property name="default">
         <bool>false</bool>
        </property>
        <property name="flat">
         <bool>false</bool>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
